src/components/title/_fields/custom-url/custom-url-fields.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import { TextField } from '@folio/stripes/components';
+
+export default class CustomUrlFields extends Component {
+  validate(value) {
+    let errors;
+
+    if (value && value.length > 600) {
+      errors = <FormattedMessage id="ui-eholdings.validate.errors.customUrl.length" />;
+    }
+
+    if (value && value.search(/http?[s]?:\/\//g)) {
+      errors = <FormattedMessage id="ui-eholdings.validate.errors.customUrl.include" />;
+    }
+
+    return errors;
+  }
+
+  render() {
+    return (
+      <div>
+        <div data-test-eholdings-custom-url-textfield>
```

Comment:
Is there a structural reason for these nested `div`s?

Reply:
There isn't. Removed. I should do another pass on `<Field>`s to see if we can put `data-test-*` attributes on those to render on their children `stripes-components`. We have a lot of `<div>`s whose sole purpose is to apply a `data-test-*`.